index.js

```diff
@@ -48,9 +49,15 @@ function factory(prng) {
     return encodeTime(seedTime, TIME_LEN) + encodeRandom(RANDOM_LEN)
   }
 
+  function lower(seedTime) {
+    ENCODING = ENCODING_LOWER
```

Comment:
Haha, dammit. My goal was to create a pull request before I get to the final station on my train. I achieved it, but I guess the code just had to be buggy 🙈